test/cmd/lfstest-gitserver.go

```diff
@@ -76,6 +76,14 @@ func main() {
 			return
 		}
 
+		if strings.Contains(r.URL.Path, "/info/lfs/locks") {
```

Comment:
I used these to make sure all requests were routed correctly: https://github.com/git-lfs/git-lfs/pull/1818/files#diff-a3b6c1ff24bf259401c990b87d408bb6R160.